ciao-launcher/vmconfig.go

```diff
@@ -96,6 +96,14 @@ func (cfg *vmConfig) findVolume(UUID string) *volumeConfig {
 	}
 	return nil
 }
+func (cfg *vmConfig) haveBootableVolume() bool {
```

Comment:
It does work as is.  But yes the ordering varies depending on how things are defined.  I can define things in a way that there is or is not a scratch volume on the instance add command line.  If the scratch is first it comes up as /dev/vda in the guest and the image is /dev/vdb and the guest still boots.  Without the scratch volume the image is /dev/vda.  Or I can do no image in the workload_storage.csv and mix around the ordering of -volume on the cli and move the disks around arbitrarily.  To me this mimics real life disk discovery in a computer.  I can see a rationale for enforcing a specific ordering and that seems to be what boot_index is about.  Launcher would need to look at the list of volumes any optional boot_indexes and hand the volumes to qemu in the index order to get them to come up with names in that order.